### PR TITLE
ZCS-9101 : Adding Chartjs package to load in portal.

### PR DESCRIPTION
**Pull request by BhavinBathani**

### CODE REVIEW

File: WebRoot/js/ajax/package/Chartjs.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Web Client
+ * Copyright (C) 2006, 2007, 2008, 2009, 2010, 2011, 2013, 2014, 2015, 2016, 2017, 2018 Synacor, Inc.
```
Comment (silentsakky) [inline WebRoot/js/ajax/package/Chartjs.js]: till 2020

File: WebRoot/js/ajax/package/Chartjs.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * for the Original Developer. In addition, Exhibit A has been modified to be consistent with Exhibit B.
+ *
+ * Software distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTY OF ANY KIND, either express or implied.
+ * See the License for the specific language governing rights and limitations under the License.
+ * The Original Code is Zimbra Open Source Web Client.
+ * The Initial Developer of the Original Code is Zimbra, Inc.  All rights to the Original Code were
+ * transferred by Zimbra, Inc. to Synacor, Inc. on September 14, 2015.
+ *
+ * All portions of the code are Copyright (C) 2006, 2007, 2008, 2009, 2010, 2011, 2013, 2014, 2015, 2016, 2017, 2018 Synacor, Inc. All Rights Reserved.
```
Comment (silentsakky) [inline WebRoot/js/ajax/package/Chartjs.js]: same as above

File: WebRoot/js/ajax/package/Chartjs.js (hunk 1 above)
Comment (BhavinBathani) [reply WebRoot/js/ajax/package/Chartjs.js]: Fixed.

File: WebRoot/js/ajax/package/Chartjs.js (hunk 2 above)
Comment (BhavinBathani) [reply WebRoot/js/ajax/package/Chartjs.js]: Fixed.

**Review by silentsakky — APPROVED** [review]

**Review by lavegupta — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Pull request merged**